python/lsst/meas/modelfit/integrals.cc

```diff
@@ -21,21 +21,23 @@
  * see <http://www.lsstcorp.org/LegalNotices/>.
  */
 
+#include "pybind11/pybind11.h"
 
-%include "lsst/meas/modelfit/optimizer.i"
+#include "lsst/meas/modelfit/integrals.h"
 
-%{
-#include "lsst/meas/modelfit/GeneralPsfFitter.h"
-#include "lsst/meas/modelfit/DoubleShapeletPsfApprox.h"
-%}
+namespace py = pybind11;
+using namespace pybind11::literals;
 
-%include "lsst/meas/modelfit/GeneralPsfFitter.h"
-%include "lsst/meas/modelfit/DoubleShapeletPsfApprox.h"
+namespace lsst {
+namespace meas {
+namespace modelfit {
+namespace {
 
-%pythoncode %{
-import lsst.pex.config
+PYBIND11_PLUGIN(integrals) {
+    py::module mod("integrals");
+    mod.def("phid", &detail::phid);
+    mod.def("bvnu", &detail::bvnu);
```

Comment:
Ouch. I had not even noticed that the entire contents of `integrals.h` is defined in sub-namespace `detail`. I am very uncomfortable with that cheat, especially since the `detail` directory exists and `integrals.h` could easily be moved there.

If you are willing to clean that up I suggest moving the C++ header file (it is only included by 4 files, all of them in meas_multifit) and making a `detail` subdirectory in the python dir.